ironfish/src/rpc/routes/event/onTransactionGossip.test.ts

```diff
@@ -28,7 +28,6 @@ describe('Route event/onTransactionGossip', () => {
       serializedTransaction: transaction.serialize().toString('hex'),
     })
 
-    response.end()
-    expect(response.status).toEqual(200)
```

Comment:
Why was this assertion removed?

Reply:
I think what this line was trying to do was tell the server that its done listening to the stream endpoint. With an HTTP or TCP connection the client can do that by just closing the connection. When the client/server is in memory however it can't just close the connection and we had no way of telling the RPC handler to stop the stream. That's what the `response.close()` function does.

Reply:
Got it, thanks for the detailed explanation